packages/components/e2e/component-objects/src/main/java/org/talend/component/Component.java

```diff
@@ -37,20 +43,22 @@ public Component(WebDriver driver, String name, String selector) {
         this.jsExec = (JavascriptExecutor) driver;
         this.name = name;
         this.selector = selector;
+        this.wait = new WebDriverWait(this.driver, 1);
     }
 
     /**
      * Component constructor
      *
      * @param driver Selenium WebDriver
-     * @param root Component root element
+     * @param root   Component root element
      */
     public Component(WebDriver driver, String name, WebElement root) {
         LOGGER.info("Component " + name + " from provided root");
         this.driver = driver;
         this.jsExec = (JavascriptExecutor) driver;
         this.name = name;
         this.root = root;
+        this.wait = new WebDriverWait(this.driver, 7);
```

Comment:
should be passed as parameter

Reply:
Why 7 ?

Reply:
No wait method should be used. 
It prevents reproducible builds and introduces latency.

Reply:
This introduce latency only if you use it, the goal is precisely to the latency. But this was already discussed with Jimmy who thinks that this is not the place to introduce it, but it is better to introduce the wait only when a component needs it. So this PR is useless